ESP32main.ino

```diff
@@ -222,65 +236,61 @@ void enviar(void *parameter){
       switch (state_env){
       case 0:
               // No hacer nada de nada de res de res
-        break;  
+        break;
       case 1: { /*Enviar volumen*/
-        //Assignar volumen a la variable 
+        //Assignar volumen a la variable
         vTaskResume(esp32Client);
-        state_env = 2;
+        state_env = 2; //error
       }
-        break;      
-      case 2: { /*Esperar*/
-              
+        break;
+      case 2: { /*Esperar*/ //error
```

Comment:
This isn't correct. Inside the switch case, the state_env is checked again. This thread must be called when needed from FFT thread so ideally, the conditions expected for "enviar" thread to work are previously met.

@olgaRuiz7 @LauLax